api/AudioParamMap.json

```diff
@@ -14,7 +14,7 @@
             "version_added": "≤79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "76"
           },
           "firefox_android": {
             "version_added": null
```

Comment:
Can you make a guess for firefox_android too, here and elsewhere that you're updating only firefox?

Reply:
I left it `null` since I was waiting on the PR to add Fenix (FxA 79), but now that it's merged I can do the mirroring!